tutorials/automated-publishing-container-builder/index.md

```diff
@@ -0,0 +1,163 @@
+
+Specify Main page as "index.html" and Save:
+
+![Specify main page](https://storage.googleapis.com/gcp-community/tutorials/automated-publishing-container-builder/configure-website.png)
+
+Now, configure your domain name’s DNS records to [create a CNAME
+record][gcs-hosting] that points to the Google Cloud Storage. This makes clients
+requesting your website point to the Google Storage APIs.
+
+## Set up automated builds
+
+We will use [Google Cloud Container Builder][gcb] and the [Build Triggers][bt]
+feature to make sure we upload our website automatically every time we push a
+new git commit to our source repository.
+
+> Note: If you do not have a repository on GitHub, you can fork [this sample
+> repository][sample-repo] for the purposes of this tutorial.
+
+Head over to the Container Registry &rarr; [Build Triggers][p6n-triggers]
+section on Google Cloud Console and click “Add trigger”:
+
+![Add build trigger on Container Registry section](https://storage.googleapis.com/gcp-community/tutorials/automated-publishing-container-builder/add-trigger-button.png)
+
+Then select GitHub as the source of your repository. In the next screen you may
+be asked to authorize access to your GitHub account and repositories. This is
+needed for Google Cloud Source Repositories to mirror and create commit hooks on
+your GitHub repositories.
+
+![Select GitHub as the source](https://storage.googleapis.com/gcp-community/tutorials/automated-publishing-container-builder/select-source.png)
+
+Then, pick your repository from the list. If you forked the sample repository
+above, pick it here:
+
+![Select the Git repository](https://storage.googleapis.com/gcp-community/tutorials/automated-publishing-container-builder/select-repo.png)
+
+In the next screen
+
+- give this Build Trigger a name (e.g. publish-website)
+- choose Build Configuration "cloudbuild.yaml"
+- choose Trigger Type ”Branch”
+- type cloudbuild.yaml to the file location
+
+![Create build trigger](https://storage.googleapis.com/gcp-community/tutorials/automated-publishing-container-builder/create-trigger.png)
+
+Now, create a `cloudbuild.yaml` file with the following contents on your
+repository. Note that you can add files to your repository on GitHub website, or
+by cloning the repository on your development machine:
+
+```yaml
+steps:
+  - name: gcr.io/cloud-builders/gsutil
+    args: ["-m", "rsync", "-r", "-c", "-d", "./vision/explore-api", "gs://hello.alp.im"]
+```
+
+This YAML file declares a build step with `gsutil -m rsync` command and makes
+sure that the website is uploaded to the storage bucket. The `-m` flag
+accelerates upload by processing multiple files in parallel and the `-c` flag
+avoids re-uploading unchanged files.
+
+The last command in the `args` is the name of your storage bucket prefixed with
+`gs://`.
+
+In our sample repository, we wanted to upload `vision/explore-api/` directory,
```

Comment:
Move this paragraph before the previous paragraph, and replace the beginning with: "If you're using the sample repository, you should upload the directory `./vision/explore-api`. If you would like..."